sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/store/KittenStoreImpl.kt

```diff
@@ -1,66 +1,54 @@
 package com.badoo.reaktive.samplemppmodule.store
 
-import com.badoo.reaktive.disposable.CompositeDisposable
-import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.scope.DisposableScope
 import com.badoo.reaktive.observable.Observable
 import com.badoo.reaktive.samplemppmodule.store.KittenStore.Intent
 import com.badoo.reaktive.samplemppmodule.store.KittenStore.State
 import com.badoo.reaktive.scheduler.computationScheduler
 import com.badoo.reaktive.scheduler.mainScheduler
 import com.badoo.reaktive.single.map
 import com.badoo.reaktive.single.observeOn
-import com.badoo.reaktive.single.subscribe
 import com.badoo.reaktive.subject.behavior.behaviorSubject
 import com.badoo.reaktive.utils.ensureNeverFrozen
 
 internal class KittenStoreImpl(
     private val loader: KittenLoader
-) : KittenStore {
+) : KittenStore, DisposableScope by DisposableScope() {
 
-    private val _states = behaviorSubject(State()).ensureNeverFrozen()
+    private val _states = +behaviorSubject(State()).ensureNeverFrozen()
```

Comment:
`+behaviorSubject(State())` feels like Compose...
Can't say yet, is it good or not.

Reply:
No good, they got rid of it :)

Reply:
I can remove. But it is taken from the official docs and looks nice to me) https://kotlinlang.org/docs/reference/type-safe-builders.html

Reply:
Personally I would prefer a function call instead of the plus operator.
Plus operator isn't named, so the reader has to be actively aware of the context where it is called (it does different things in different places). I think plus operator works better in places where it is clear what you are adding to. Having to call scope at least tells you what is happening.
(particularly as people are likely reading the examples on github, and can't look into the definition of what + is adding to).

Reply:
Ok, looks like nobody like it. Will remove and use `.scope()` instead.

Reply:
Removed